packages/sd-jwt/__tests__/verify.test.ts

```diff
@@ -0,0 +1,98 @@
+import { it, expect, describe, afterAll } from "vitest";
+import { verify } from "../src";
+import * as fs from "fs";
+import * as http from "http";
+import * as path from "path";
+
+// EXAMPLE from https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-04.html#name-example-1-sd-jwt
+const sdjwt_header_and_payload =
```

Comment:
If it's a variable in snake case maybe all caps is better?

Reply:
I see camelCase is a de-facto convention on js, I did enforce it a lot in the past.

I was kinda experimenting using snake_case, as it allows a more discursive variable naming

```ts
sdjwt_with_disclosures_signed_with_key_a

sdjwtWithDisclosuresSignedWithKeyA
```

You think we could give it a try?

Reply:
I think it's a great approach. My suggestion was to use SNAKE_KASE in uppercase `SDJWT_WITH_DISCLOSURES_SIGNED_WITH_KEY_A`